packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -36,18 +30,25 @@ export interface Relay {
   instantRelayFeePct: string;
   quoteTimestamp: number;
   realizedLpFeePct: string;
+  priceRequestTime: number;
   depositHash: string;
-  relayState: RelayState;
+  relayState: ClientRelayState;
   relayHash: string;
 }
 
 export interface InstantRelay {
   instantRelayer: string;
 }
 
+export interface BridgePoolData {
+  contract: BridgePoolWeb3;
+  currentTime: number;
+  relayNonce: number;
+}
+
 export class InsuredBridgeL1Client {
   public readonly bridgeAdmin: BridgeAdminInterfaceWeb3;
-  public bridgePools: { [key: string]: BridgePoolWeb3 }; // L1TokenAddress=>BridgePoolClient
+  public bridgePools: { [key: string]: BridgePoolData }; // L1TokenAddress=>BridgePoolData
```

Comment:
Because we need the Relayer to know about the current contract time and relay nonce when instant relaying (slow + speedupRelay) since `speedUpRelay` takes the `RelayData` as a function param. So, instead of mapping L1 tokens to just the bridgepool contracts I thought it best to map to the contract, + those two state values mentioned. WDYT?